### PR TITLE
test: Drop obsolete pybridge expected log messages

### PR DESCRIPTION
**Pull request by martinpitt**
These got addressed by recent bug fixes.

### CODE REVIEW

**Comment by martinpitt** [comment]
[Log without any extra filtering](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18235-20230124-154330-1b4e99ef-fedora-37-pybridge/log.html) --  24 failures. I put back the ones which still apply.

**Review by allisonkarlitskaya — APPROVED** [review]
Very nice!!

Happy to see this land, assuming it's green.

File: test/common/testlib.py [hunk 1]
```diff
@@ -1801,27 +1801,14 @@ def check_journal_messages(self, machine=None):
                 r"File .*asyncio/events.py.*",
                 r"self._context.run\(self._callback, \*self._args\)",
                 r"File .*cockpit/transports.py.* _read_ready",
-                r"data = os.read\(self._in_fd, _Transport.BLOCK_SIZE\)",
-                r"ConnectionResetError: \[Errno 104\] Connection reset by peer"]
+                r"data = os.read\(self._in_fd, _Transport.BLOCK_SIZE\)"]
```
Comment (allisonkarlitskaya) [inline test/common/testlib.py:1804]: *mumbles something about trailing commas*

**Comment by martinpitt** [comment]
Nice, turns out it did get green! Triggering fcos.

File: test/common/testlib.py (hunk 1 above)
Comment (martinpitt) [reply test/common/testlib.py:1804]: *mumbles agreement, but is too lazy to re-push/re-run again*

We need to figure out the remaining issues anyway, at least this won't live forever..

**Pull request merged**